kotlin/internal/toolchains.bzl

```
@@ -268,7 +268,7 @@ _kt_toolchain = rule(
             default = Label("//third_party:empty.jdeps"),
         ),
         "jacocorunner": attr.label(
-            default = Label("@bazel_tools//tools/jdk:JacocoCoverage"),
+            default = Label("@remote_java_tools//:jacoco_coverage_runner"),
```

```suggestion
            default = Label("@@rules_java++toolchains+remote_java_tools//:jacoco_coverage_runner"),
```
maybe this way it can finds it?